xc/domain.ml

```diff
@@ -202,7 +202,7 @@ let make ~xc ~xs vm_info uuid =
     let zeroperm = Xenbus_utils.rwperm_for_guest 0 in
     debug "VM = %s; creating xenstored tree: %s" (Uuid.to_string uuid) dom_path;
 
-    let create_time = Oclock.gettime Oclock.monotonic in
```

Comment:
Documentation says:
```
Only use Mtime_clock.now if you need inter-process time correlation, otherwise prefer Mtime_clock.elapsed and counters.
```
I believe this this the correct clock but wanted to double check.